bridge/server/package.json

```diff
@@ -24,7 +24,6 @@
     "express-session": "1.17.3",
     "helmet": "5.1.1",
     "mongodb": "4.10.0",
-    "morgan": "1.10.0",
```

Comment:
It's not removed in the lock file right?
Could you remove these dependencies via `yarn remove morgan @types/morgan`? Then yarn will take care that the package.json and the lock file is corrected